src/query/epa/epa3.rs

```diff
@@ -327,13 +344,15 @@ impl EPA {
             if face.deleted {
                 continue;
             }
-
-            let cso_point = CSOPoint::from_shapes(pos12, g1, g2, &face.normal);
+            let face_normal = &face
+                .normal
```

Comment:
do the same logic as https://github.com/Vrixyz/parry/pull/4/files#diff-f1b840a30d9b3ff6e26f4a912afabd554d9c74bd6206707f864e9f9815bf24f0R312

+ remove deleted face ?